doc/format-run.md

```diff
@@ -185,24 +185,28 @@ whole key per page: we can typically store just 4 bytes (32bits) per page. This
 is a factor of 8 saving for 32 byte keys.
 
 The representation consists of
-1. the number of range finder bits (0..16)
+1. a primary array of 32bit words, one entry per page in the index
 2. a range finder array, of 2^n+1 entries of 32bit each (n = range finder bits)
-3. a primary array of 32bit words, one entry per page in the index
-4. a clash indicator bit vector, one bit per page in the index
+3. a clash indicator bit vector, one bit per page in the index
+4. a larger-than-page indicator bit vector, one bit per page in the index
 5. a clash map, mapping each page with a clash indicator to the full minimum
    key for the page
+6. the number of range finder bits (0..16) (as a 8bit value)
+7. the number of pages in the primary array (as a 64bit value)
+8. the number of keys in the corresponding key/ops file (as a 64bit value)
```

Comment:
What would we use this number for? I think we only need to know the number of pages

Reply:
It's not strictly necessary for the index format itself, but we were discussing the other day that we wanted to know this number, and we needed somewhere convenient to put it.